TypeEquality/Teq.fsi

```diff
@@ -91,17 +91,20 @@ module Teq =
         val domain<'domain1, 'domain2, 'range> : Teq<'domain1, 'domain2> -> Teq<'domain1 -> 'range, 'domain2 -> 'range>
 
         /// Given a type equality between two function types, returns the type equality on their corresponding domains.
-        val domainOf<'domain1, 'domain2, 'range1, 'range2> : Teq<'domain1 -> 'range1, 'domain2 -> 'range2> -> Teq<'domain1, 'domain2>
+        val domainOf<'domain1, 'domain2, 'range1, 'range2> : Teq<'domain1 -> 'range1, 'domain2 -> 'range2>
```

Comment:
I think I mildly prefer the following style:

```fsharp
val domainOf<'a, 'b, 'r1, 'r2> :
    Teq<'a -> 'r1, 'b -> 'r2>
    -> Teq<'a, 'b>
    -> Teq<'r1, 'r2>
```

But I don't mind enough to stop this PR going in.

Reply:
Created https://github.com/fsprojects/fantomas/issues/1181 to follow-up on that.
I think there is no real reason here why Fantomas formatted it that way.